packages/docs/GLOSSARY.md

```diff
@@ -176,7 +176,7 @@ A device or computer running the Celo Blockchain software that keeps typically o
 
 [CELO](#celo-native-asset) balances held in escrow at the Locked Gold contract for the account that deposited it there. This permits that balance to be used for voting in validator elections, governance proposals, and to meet staking requirements for registering a validator or validator group.
```

Comment:
Consider just calling this anchor #CELO. My understanding of the long-form name is that it's useful when just using CELO would be ambiguous. E.g. Stablecoins on the Celo platform are backed by the Celo native asset.

Reply:
Also, consider mentioning here that CELO was previously called Celo Gold (cGLD).

Reply:
This is now addressed in the paragraph below. There is an entry on CELO, one on Celo Gold, and another on Celo Native Asset -- I think they should be sufficient.